pkg/dao/interfaces.go

```diff
@@ -134,6 +134,7 @@ type MetricsDao interface {
 	PendingTasksAverageLatency(ctx context.Context) float64
 	PendingTasksCount(ctx context.Context) int64
 	PendingTasksOldestTask(ctx context.Context) float64
+	RHReposNoSuccessfulSnapshotTaskIn36Hours(ctx context.Context) int64
```

Comment:
```suggestion
	RHReposSnapshotNotCompletedInLast36HoursCount(ctx context.Context) int64
```
This is a nitpick, but what do you think about renaming this metric?

"No" can also be an abbreviation for "number", so "RHReposNoSuccessfulSnapshotTaskIn36Hours" could be interpreted as "Number of sucessful snapshot tasks"

Reply:
Haven't though of that interpretation, changed! 😁👍🏼